server.go

```diff
@@ -123,6 +134,25 @@ func (server *Server) Run() error {
 		server.ServeTLS()
 	}
 
+	ticker := time.NewTicker(50 * time.Second)
+
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				server.errorsMutex.Lock()
+				server.Log.Warnf("Start DUMP")
+				for remoteAddr, str := range server.errors {
+					if str.bad > 10 || str.goodTls > 100 {
```

Comment:
is this dumping temporary? I guess `server.errors` is a subject to grow during the time (on long run) and might take some memory :)

Reply:
nit: just in case, if `servers.errors` is not supposed to be purged, that makes sense to make a copy of the map before counting+dumping and release the lock to allow the main flow keep working.